keywords/SyncGateway.py

```diff
@@ -226,6 +227,15 @@ def start_sync_gateways(self, cluster_config, url=None, config=None):
             playbook_vars["autoimport"] = '"import_docs": "continuous",'
             playbook_vars["xattrs"] = '"enable_shared_bucket_access": true,'
 
+        if is_cbs_ssl_enabled(cluster_config) and get_sg_version(cluster_config) >= "1.5.0":
```

Comment:
This is a string comparison. Have you tested this code for other SG versions? You might want to checkout compare_versions(version_one, version_two): in keywords.utils

Reply:
Yes, I tested this 1.4 and made sure it works. I tested with 1.4 ssl enabled and disabled